src/client/index.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { Router, match, browserHistory } from 'react-router'
+import { match, browserHistory } from 'react-router'
 import AsyncProps from 'async-props'
 import 'location-origin'
 import 'es6-promise/auto'
 import RouteWrapper from '../shared/route-wrapper'
 import config from 'tapestry.config.js'
 import mitt from 'mitt'
+import { AppContainer } from 'react-hot-loader'
+import Root from './root'
```

Comment:
Root just wraps our `Router` component, which we need to import from a separate file to get Hot Reloading working.